packages/core/src/editor/Block.css

```diff
@@ -204,9 +206,11 @@ NESTED BLOCKS
 }
 
 /* Unordered */
-.bn-block-content[data-content-type="bulletListItem"] {
+.bn-block-content[data-content-type="bulletListItem"]::before {
```

Comment:
can you explain why this changed? why the before element that wasn't here earlier?

Reply:
Ah I forgot to mention this in the PR description - I adjusted offsets for list items, indented blocks, and the "add block" button of the toggle blocks to make them all the same. So all of these are now vertically in-line now, and these styles are a part of that.